Xamarin.Forms.Platform.UAP/SwitchRenderer.cs

```diff
@@ -28,6 +36,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Switch> e)
 			}
 		}
 
+		private void Control_Loaded(object sender, RoutedEventArgs e)
```

Comment:
This has to be set when the control is fully loaded, else it will crash/will not have any effect

Reply:
Could yo please drop `private` and rename to `OnControlLoaded`.

Reply:
whoops... Done